scripts/actions/spells/trust/tenzen_ii.lua

```diff
@@ -13,6 +13,16 @@ end
 
 spellObject.onMobSpawn = function(mob)
     xi.trust.message(mob, xi.trust.messageOffset.SPAWN)
+	
+	mob:addSimpleGambit(ai.t.TARGET, ai.c.ALWAYS, 0, ai.r.RATTACK, 0, 0)
```

Comment:
this should (usually) be off for ranged-only attackers or else they will additionally do an un-animated melee attack when in range for it.